add "bind" to validResourceVerbs in "create role" cli

CaoShuFeng opened the pull request:
"bind" and "impersonate" are also a legal verbs according this:
https://github.com/kubernetes/kubernetes.github.io/pull/2771/files
**Release note**:
<!--  Steps to write your release note:
1. Use the release-note-* labels to set the release note state (if you have access) 
2. Enter your extended release note in the below block; leaving it blank means using the PR title as the release note. If no release note is required, just write `NONE`. 
-->
```NONE
```

k8s-reviewable commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/42612)
<!-- Reviewable:end -->

CaoShuFeng committed:
add "bind" and "impersonate" to legal verbs

"bind" and "impersonate" are also legal verbs according this:
https://github.com/kubernetes/kubernetes.github.io/pull/2771/files

dims commented:
@k8s-bot ok to test

CaoShuFeng commented:
@k8s-bot non-cri e2e test this

CaoShuFeng commented:
@k8s-bot gci gce e2e test this

CaoShuFeng commented:
@k8s-bot non-cri e2e test this

CaoShuFeng commented:
@k8s-bot non-cri e2e test this

dims commented:
/approve

liggitt commented on pkg/kubectl/cmd/create_role.go:
```diff
@@ -42,10 +42,10 @@ var (
 
 	// Valid resource verb list for validation.
-	validResourceVerbs = []string{"*", "get", "delete", "list", "create", "update", "patch", "watch", "proxy", "redirect", "deletecollection", "use"}
+	validResourceVerbs = []string{"*", "get", "delete", "list", "create", "update", "patch", "watch", "proxy", "redirect", "deletecollection", "use", "bind"}
```
add "impersonate" as well (c.f. https://github.com/kubernetes/kubernetes.github.io/pull/2771/files)

CaoShuFeng replied:
Done.
Thanks for your review.

liggitt commented:
go ahead and squash, then LGTM

CaoShuFeng commented:
@k8s-bot kops aws e2e test this

CaoShuFeng commented:
@k8s-bot kops aws e2e test this

CaoShuFeng commented:
@k8s-bot kops aws e2e test this

CaoShuFeng commented:
@k8s-bot kops aws e2e test this

dims commented:
/approve
/lgtm

CaoShuFeng commented:
rebased.

CaoShuFeng commented:
ping.

liggitt commented:
/assign

liggitt commented:
/lgtm
